index.html

```diff
@@ -42,5 +42,175 @@ <h1 class="name"> Hey there <br>
+            <ul class="text">
+              <li class="text-ruby">Ruby on Rails</li>
+              <li class="text-css">CSS</li>
+              <li class="text-java">JavaScript</li>
+              <li class="text-html">HTML</li>
+            </ul>
+            <div class="button1"><button class="button-text" type="button">See project </button></div>
+          </div>
+      </div>
+      </section>
+        <section class="about-container">
+          <div class="text-aline">
+            </div>
+            <div class="text-about">
+              <div class="plane-container">
+                <h3>About me</h3>
+                <div class="plane-image">
+                  <img src="Images 2/shape.svg" alt="plane">
+                </div>
+              </div>
+              <div class="test2">
+                <div>
+                  <p class="h6"> I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hesitate to contact me.</p>
+                </div> 
+              </div>
+                <div class="button2">
+                  <button class="button-resume" type="button">Get my resume </button>
+                </div>
+            </div>
+            <div class="plane">
+              <img src="Images 2/shape2.svg" alt="square">
+            </div>
+          </div>
+           <div class="section6">
+            <article class="qoute">
+              <div class="holder-text">
+                <img src="Images 2/placeholder2.svg" alt="holder">
+                <div class="holder-text1">
+                  <img src="Images 2/language-icon.svg" alt="holder">
+                  <ul>
+                    <li class="Language">
+                      <p>Languages</p>
+                      </li>
+                        <ul class="media">
+                          <li>JavaScript</li>
+                          <li>Ruby</li>
+                          <li>HTML</li>
+                          <li>CSS</li>
+                        </ul>
+                  </ul>              
+                </div>
+              </div>
+              </article>
+                <section class="qoute">
+                    <div class="holder-text">
+                    <img src="Images 2/placeholder2.svg" alt="holder">
+                    <div class="holder-text1">
+                      <img src="Images 2/frame-icon.svg" alt="holder">
+                      <ul>
+                        <li class="Language">
+                          <p>Frameworks</p>
+                        </li>
+                        <ul class="media">
+                          <li>React.js</li>
+                          <li>Ruby on Rails</li>
+                          <li>RSpec</li>
+                          <li>Capybara</li>
+                          <li>Selenium</li>
+                        </ul>
```

Comment:
this list also needs  a bit of horizontal margin refer to the images

![image (3)](https://github.com/Wils-FOURR-speed/Portfolio-setup-and-mobile-first-project/assets/59707859/6c410c74-1fed-4679-888f-8e3d717733f8)
![image (4)](https://github.com/Wils-FOURR-speed/Portfolio-setup-and-mobile-first-project/assets/59707859/25440f8e-0b5b-4ac4-83bc-c16ecc72eab3)